sway-fmt-v2/CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# Contributing to the Sway Language Formatter
+
+Firstly, thank you for taking interest in advancing the Sway language formatter! This guide will walk you through installation and best practices for contributing to this project.
+
+## Installation
+
+In order to see `swayfmt` in action you will need both `swayfmt` and the [`forc-fmt`](../forc-plugins/forc-fmt/) plugin. As previously stated, you can install these via [`fuelup`](https://github.com/FuelLabs/fuelup) which will install the complete Fuel toolchain, or follow the below instructions to install from source.
+
+**note:** `cargo` is a pre-requisite to this build
+
+1. move to your preferred directory
+2. clone the Sway repo
+3. build from manifest and move the compiled result to your `.cargo/bin` folder
+
+```sh
+git clone https://github.com/FuelLabs/sway.git
+cargo build --manifest-path ~/sway/forc-plugins/forc-fmt/Cargo.toml
+mv ~/sway/target/debug/forc-fmt ~/.cargo/bin
+```
+
+### Testing
+
+Simply run the `forc-fmt` command on the file you want to format:
```

Comment:
This is not guarenteed to work because it is dependent upon the location from which the command is executed. I think the doc should outline where the command can be run e.g. can it be run inside the `src` directory, can it be run on the same level as the `src`, what happens if you run it outside of that project? A beginner might not know and I sometimes run the command from an incorrect location and an error pops up telling me that the manifest file cannot be located - which is not a great error.